.github/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# This will test fluence with two jobs.
+# We choose jobs as they generate output and complete, and pods
+# are expected to keep running (and then would error)
+
+set -eEu -o pipefail
+
+# ensure upstream exists
+# This test script assumes fluence image and sidecar are already built
+make prepare
+
+# Keep track of root directory to return to
+here=$(pwd)
+
+# Never will use our loaded (just built) images
+cd upstream/manifests/install/charts
+helm install \
+  --set scheduler.image=ghcr.io/flux-framework/fluence:latest \
+  --set scheduler.sidecarimage=ghcr.io/flux-framework/fluence-sidecar:latest \
+  --set scheduler.pullPolicy=Never \
+  --set scheduler.sidecarPullPolicy=Never \
+    schedscheduler-plugins as-a-second-scheduler/
+
+# These containers should already be loaded into minikube
+echo "Sleeping 10 seconds waiting for scheduler deploy"
+sleep 10
+kubectl get pods
+
+# This will get the fluence image (which has scheduler and sidecar), which should be first
+fluence_pod=$(kubectl get pods -o json | jq -r .items[0].metadata.name)
+echo "Found fluence pod ${fluence_pod}"
+
+# Show logs for debugging, if needed
+echo
+echo "⭐️ kubectl logs ${fluence_pod} -c sidecar"
+kubectl logs ${fluence_pod} -c sidecar
+echo
+echo "⭐️ kubectl logs ${fluence_pod} -c scheduler-plugins-scheduler"
+kubectl logs ${fluence_pod} -c scheduler-plugins-scheduler
+
+# We now want to apply the examples
+cd ${here}/examples/test_example
+
+# Apply both example jobs
+kubectl apply -f fluence-job.yaml
+kubectl apply -f default-job.yaml
```

Comment:
Note that scheduling pods with kube-scheduler and Fluence on the same cluster isn't supported. There isn't currently any way to propagate pod-to-node mappings generated by kube-scheduler to Fluence.

It's important that `kubectl apply -f fluence-job.yaml` is executed before `kubectl apply -f default-job.yaml`, and that they don't specify limits or requests so they could be scheduled on the same node. That's currently the case in this PR, but I'm emphasizing it for posterity. 

Regardless, there still may be some funky race condition that occurs and results in unschedulable pods.

Reply:
> It's important that kubectl apply -f fluence-job.yaml is executed before kubectl apply -f default-job.yaml, and that they don't specify limits or requests so they could be scheduled on the same node. That's currently the case in this PR, but I'm emphasizing it for posterity.

Gotcha - I think likely for the testing cluster (and the example we already had in main) we are just doing that, putting them on the same node, and since it's a tiny kind or otherwise local cluster, there hasn't been an issue. If we extended this to an actual setup, there would be. This is an important point and I've opened an issue for emphasizing it in in future docs: https://github.com/flux-framework/flux-k8s/issues/53 and maybe we can think of a creative way to allow for both, possibly with kueue resource flavors that create distinct (separate) resources that are labeled for each.